item.rb

```diff
@@ -1,36 +1,60 @@
 require 'date'
+require './label_methods'
+require 'securerandom'
 
 class Item
-  attr_reader :id
-  attr_accessor :genre, :label, :published_date :archived
+  attr_reader :id, :author
+  attr_accessor :genre, :source, :label, :publish_date, :archived
 
-  def initialize(id:, published_date: archived: false)
-    @id = :id
+  def initialize(id:, publish_date:, archived: false)
```

Comment:
According to our last team meeting, the idea was to avoid using the structure from the last project to have a simpler structure, working with objects directly so we don't need to use the id as an optional parameter.

This change will break the existing author feature already reviewed and pushed in #36, so please try to keep in mind the item feature will be shared by the entire team before changing the structure for it.

Another thing is that please consider avoiding using symbols to initialize variables other than when having optional boolean parameters. It makes the syntax more complex (having to initialize functions with `item = Item.new(published_date:'xx-xx-xxxx')` instead of just `item = Item.new('xx-xx-xxxx')` and I don't see a gain for the added complexity and breaking the existing pushed features(like author I just mentioned). 

Can you share why you prefer this syntax?